src/pages/index.js

```diff
@@ -65,35 +86,128 @@ profileEditButton.addEventListener("click", () => {
   editPopupForm.open();
 });
 
+//Avatar Profile change
+avatarEditButton.addEventListener("click", () => {
+  avatarFormValidator.toggleButtonState();
+  avatarProfilePopupForm.open();
+});
+
 /*Functions*/
 /*Cards*/
 function renderCard(cardData) {
-  const card = new Card(cardData, "#card-template", handleCardImageClick);
-  section.addItem(card.getCardElement());
+  const card = new Card({
+    data: cardData,
+    templateSelector: "#card-template",
+    handleCardImageClick,
+    myUserId: userId,
+    handleDeleteCard,
+    handleLikeButton: (cardId) => {
+      if (card.isLiked()) {
+        api
+          .removeLike(cardId)
+          .then((data) => {
+            card.updateLikes(data.likes);
+          })
+          .catch((err) => {
+            console.log(err);
+          });
+      } else {
+        api
+          .addLike(cardId)
+          .then((data) => {
+            card.updateLikes(data.likes);
+          })
+          .catch((err) => {
+            console.log(err);
+          });
+      }
+    },
+  });
+  return card.getCardElement();
+}
+let cardList;
+api
+  .getInitialCards()
+  .then((initialCards) => {
+    cardList = new Section(
+      {
+        items: initialCards,
+        renderer: (data) => {
+          const newCard = renderCard(data);
+          cardList.addItem(newCard);
+        },
+      },
+      cardsWrap
+    );
+    cardList.renderItems();
+  })
+  .catch((err) => console.log(err));
+
+let userId;
+api
+  .getUserInfo()
+  .then((formData) => {
+    userId = formData._id;
+    userInfo.setProfileAvatar(formData.avatar);
+    userInfo.setUserInfo(formData.name, formData.about);
+  })
+  .catch((err) => console.log(err));
+
+function handleEditProfileSubmit({ nameInfo, jobInfo }) {
+  api
```

Comment:
* According to the task you need to change submit buttons text (`Saving…`) while waiting for the server response. And the user should see it.